NuRadioMC/EvtGen/generator.py

```diff
@@ -726,6 +729,9 @@ def generate_surface_muons(filename, n_events, Emin, Emax,
         in case the data set is distributed over several files, this number specifies the id of the first file
         (useful if an existing data set is extended)
         if True, generate deposited energies instead of primary neutrino energies
+    config_file: string
```

Comment:
I don't think it is handy to force the file to reside in the same folder. One should be able to specify any path to file.

Reply:
The config_file can be a string indicating the files for SouthPole, MooresBay or InfIce or it can indicate a path to a custom file.